modules/standard/LinkedLists.chpl

```diff
@@ -203,6 +203,52 @@ record LinkedList {
      return ret;
    }
 
+  /*
+    find `e` in the list.
+
+    :arg e: The element to be checked for
+    :return: True if the item was found, otherwise returns false
+    :rtyle: `bool`
+   */
+  proc contains(e : eltType): bool {
```

Comment:
Let's go ahead and mark this method as having a `const` receiver. And while you're at it, stylistically we prefer to have colons hug the parameter name:

`proc const contains(e: eltType): bool;`